.gitignore

```diff
@@ -19,5 +116,16 @@
 *.tar.gz
 *.rar
 
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+### Maven ###
+target/
+pom.xml.tag
+pom.xml.releaseBackup
+pom.xml.versionsBackup
+pom.xml.next
+release.properties
+dependency-reduced-pom.xml
+buildNumber.properties
+.mvn/timing.properties
+.mvn/wrapper/maven-wrapper.jar
```

Comment:
I think this change is in conflict with the one in the https://github.com/quarkus-qe/quarkus-catalog-apps/pull/1 PR. For now, it's fine, but let's create a dependency to this.